app/controllers/api/site_configuration_controller.rb

```diff
@@ -16,7 +16,22 @@ def show
   def update
     authorize :site_configuration, :update?
     if @site_configuration.update(site_configuration_params)
-      render_success @site_configuration, "site_configuration.update_success"
+      if params[:site_configuration].key?(:inbox_enabled)
+        desired_state = ActiveModel::Type::Boolean.new.cast(params[:site_configuration][:inbox_enabled])
+        unless [true, false].include?(desired_state)
+          return render_error(
+            "site_configuration.invalid_inbox_enabled_value",
+            message_opts: { error_message: "Invalid value for inbox_enabled. Must be true or false." }
+          )
+        end
+        if @site_configuration.inbox_enabled == desired_state
+          Jurisdiction.update_all(inbox_enabled: desired_state)
```

Comment:
Review and consider the consequences of enabling and disabling all jurisdictions at once (may need agreement from jurisdictions, as in with external_api_state?)